asylum-connect/inreach-api
# Colin/1164471184183571/jwt implementation

## Description

Adding the middleware to the necessary routes which appear to have been removed in between merges.

- Asana ticket: https://app.asana.com/0/1132189118126148/1164471184183571

## PR Checklist



- [x] Tested by QA (@Alfredo-Moreira )

## How to Test

1. Run the catalog-api server.
2. Make a PATCH request to the http://localhost:8080/v1/organizations/:orgId WITHOUT a token
3. Observe the the request fails
4. Make POST request to http://localhost:8080/v1/organizations endpoint WITH a token
5. Observe the the request succeeds
6. Repeat with a DELETE request to the http://localhost:8080/v1/organizations/:orgId endpoint
